src/model/ThermalZone.cpp

```diff
@@ -2122,6 +2126,38 @@ namespace detail {
                               terminal.get(),
                               terminal->secondaryAirInletPort() );
             }
```

Comment:
Existing bit for `AirTerminalSingleDuctParallePIUReheat`

Reply:
https://github.com/NREL/OpenStudio/blob/050075639a5e1f0c8118184672d9f2ae7c50b926/src/model/ThermalZone.cpp#L2109-L2125